docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+            Timeout: 3
+        Resources:
+            HelloWorldFunction:
+            Type: AWS::Serverless::Function
+            Properties:
+                CodeUri: hello_world/
+                Handler: app.lambda_handler
+                Runtime: python3.9
+                Events:
+                    HelloWorld:
+                        Type: Api
+                        Properties:
+                        Path: /hello
+                        Method: get
+                    HelloWorldName:
+                        Type: Api 
+                        Properties:
+                        Path: /hello/{name}
+                        Method: get
+        Outputs:
+            HelloWorldApi:
+                Description: "API Gateway endpoint URL for Prod stage for Hello World function"
+                Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/hello/"
+    ```
+In this example we added two methods `hello_name` and `hello` next to each other. We added `Router` class that allow us to register what method should be called when specific request comes. 
+We created instance of it and added configuration with mapping between processing methods and http request method. 
+In lambda handler we call router instance get method to fetch reference to processing method (`hello` or `hello_name`) that will handle the request. 
+Finally we run this method and return result to API Gateway.
+
+Thanks to this approach, we can simplify our infrastructure configuration since we added all API Gateway path under `HelloWorldFunction` event section. 
+Downside of current approach is that we need to add custom class and with every new method or path we want to serve we need to add explicit configuration. 
+Also, we need to understand API Gateway request event structure in order to decapsulate requested path and http method. 
+In order to mitigate this downsides we use Lambda Powertools.
+
+=== "app.py"
+
+    ```python hl_lines="1 4 7 12 18"
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+=== "requirements.txt"
+
+    ```bash
+    aws-lambda-powertools
+    ```
+
+Since we used powertools api_gateway event handler we don't need to know internal event structure. All we do we instantiate `ApiGatewayResolver` class and we call it in our `lambda_handler`.
+Since we added route annotation as a decorator for our methods we can directly use parameters passed in request.
+In order for SAM to build our package correctly we specified lambda powertools package in our requirements file. 
```

Comment:
This is prerequisite, move it higher to the beginning, otherwise we already know it.

Reply:
Not sure about this, since we follow a more narrative approach in the quickstart powertools is not required to start. In first stage we don't use powertools we get there after we try multiple lambdas approach and custom Router.